README.md

```diff
@@ -59,13 +59,31 @@ file is included in this repository.
 
 ### Mapping repositories
 
-Edit [application.yaml](src/main/resources/application.yaml) to list all the origins (host + context path) for the SCM providers listed in the repos.csv, and provide a type: `[GITHUB,GITLAB,BITBUCKET_CLOUD,BITBUCKET,AZURE_DEVOPS]`.
-This will be used to split the clone url into an origin and path.
+Edit [application.yaml](src/main/resources/application.yaml) and list all the base uris for any private SCM providers used in the repos.csv.
+
+You will need to provide the base uri, type and any alternative uri that can be used to access the same SCM server.
+
+Example:
+```yaml
+moderne:
+  scm:
+    repositories:
+      - baseUri: https://bitbucket.example.com/stash
```

Comment:
Can we consolidate on URL over URI? Effectively they are the same WRT Moderne's usage and we use URL in subcommands such as `mod-config-scm-gitlab-base-urls`